elixir/apps/web/lib/web/live/policies/components.ex

```diff
@@ -1,9 +1,474 @@
+    </span>
+    """
+  end
+
+  defp condition(%{property: :current_utc_datetime, values: values} = assigns) do
+    assigns =
+      assign_new(assigns, :ranges, fn ->
+        {:ok, ranges} = Policies.Condition.Evaluator.parse_days_of_week_time_ranges(values)
+
+        ranges
+        |> Enum.reject(fn {_dow, time_ranges} -> time_ranges == [] end)
+        |> Enum.sort_by(fn {dow, _time_ranges} -> day_of_week_index(dow) end)
+      end)
+
+    ~H"""
+    <span class="flex flex-wrap space-x-1 mr-1">
+      on
+      <.intersperse_blocks>
+        <:separator>,</:separator>
+
+        <:item :for={{day_of_week, ranges} <- @ranges}>
+          <span class="ml-1 font-medium">
+            <%= day_of_week_name(day_of_week) <> "s" %>
+            <span :if={is_list(ranges) and ranges != []}>
+              <%= "(" <>
+                Enum.map_join(ranges, ", ", fn {from, to} ->
+                  "#{from} - #{to}"
+                end) <> ")" %>
+            </span>
+          </span>
+        </:item>
+      </.intersperse_blocks>
+    </span>
+    """
+  end
+
+  for {code, name} <- @days_of_week do
+    defp day_of_week_name(unquote(code)), do: unquote(name)
+  end
+
+  for {{code, _name}, index} <- Enum.with_index(@days_of_week) do
+    def day_of_week_index(unquote(code)), do: unquote(index)
+  end
+
+  defp condition_operator_option_name(:contains), do: "contains"
+  defp condition_operator_option_name(:does_not_contain), do: "does not contain"
+  defp condition_operator_option_name(:is_in), do: "is in"
+  defp condition_operator_option_name(:is_not_in), do: "is not in"
+  defp condition_operator_option_name(:is_in_day_of_week_time_ranges), do: ""
+  defp condition_operator_option_name(:is_in_cidr), do: "is in"
+  defp condition_operator_option_name(:is_not_in_cidr), do: "is not in"
+
+  def condition_form(assigns) do
+    ~H"""
+    <fieldset class="flex flex-col gap-2">
+      <legend class="text-lg mb-4">Conditions</legend>
+
+      <.remote_ip_location_region_condition_form form={@form} />
+      <.remote_ip_condition_form form={@form} />
+      <.provider_id_condition_form form={@form} providers={@providers} />
+      <.current_utc_datetime_condition_form form={@form} />
+    </fieldset>
+    """
+  end
+
+  defp remote_ip_location_region_condition_form(assigns) do
+    ~H"""
+    <fieldset class="mb-2">
+      <% condition_form = find_condition_form(@form[:conditions], :remote_ip_location_region) %>
+
+      <.input
+        type="hidden"
+        field={condition_form[:property]}
+        name="policy[conditions][remote_ip_location_region][property]"
+        id="policy_conditions_remote_ip_location_region_property"
+        value="remote_ip_location_region"
+      />
+
+      <div
+        class="cursor-pointer"
+        phx-click={
+          JS.toggle_class("hidden",
+            to: "#policy_conditions_remote_ip_location_region_condition"
+          )
+          |> JS.toggle_class("hero-chevron-down",
```

Comment:
```suggestion
          |> JS.toggle_class("hero-chevron-right",
```

Reply:
@jamilbk chevrons up and down are standard for accordion and collapsable items (see https://flowbite.com/docs/components/accordion/). Do you still want to go with right and left? I feel they are confusing as the element extends down, not to the left or right. As an option, we can also replace the sort icon to `bars-arrow-down` or use a different set of icons here.

Reply:
Ah I see, just trying to be consistent. Yeah we can replace sort icon with bars-arrow-down.

Do you think it makes sense to move the icon to the far right and bump the font size a bit to make the field sections a bit easier to identify at first glance?

Reply:
Ah I see, just trying to be consistent. Yeah we can replace sort icon with bars-arrow-down.

Do you think it makes sense to move the icon to the far right and bump the font size a bit to make the field sections a bit easier to identify at first glance?

Reply:
@jamilbk I used the largest font size that we use on our pages (this is basically page title size), if we make them even larger it starts to look ugly. The icon on the right also breaks this nice feeling that all of them are on the same horizontal line (text for each section has different with).

Let's do it like this - I will finish the functional part, we can merge it and start testing that the code is functional (and the connlib will be able to handle the error message). Then we can have a separate PR that addresses icons, borders and design overall.

Reply:
A bit more on sizes:
<img width="373" alt="Screenshot 2024-06-03 at 13 13 53" src="https://github.com/firezone/firezone/assets/1877644/e63734a2-b774-4931-81a9-5909ee996a4b">

So to make it larger we need to make page titles larger all across UX taming them from XL to 2XL.

Reply:
Kk, will do a quick design pass after merge (mostly spacing stuff otherwise LGTM!)